### PR TITLE
Update the function and tutorial of CEFCON.

### PR DESCRIPTION
**Pull request by DBinary**
Update the function and tutorial of CEFCON.

### CODE REVIEW

File: omicverse/CEFCON/CEFCON.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
+import argparse
+from os import fspath
+from pathlib import Path
+import pandas as pd
+
+import sys
+sys.path.insert(0, '/home/wangpeizhuo')
```
Comment (Starlitnightly) [inline omicverse/CEFCON/CEFCON.py]: ?

File: omicverse/CEFCON/CEFCON.py [hunk 2]
```diff
@@ -0,0 +1,114 @@
+import argparse
+from os import fspath
+from pathlib import Path
+import pandas as pd
+
+import sys
+sys.path.insert(0, '/home/wangpeizhuo')
+from cefcon.cell_lineage_GRN import NetModel
+from cefcon.cefcon_result_object import CefconResults
+from cefcon.utils import data_preparation
```
Comment (Starlitnightly) [inline omicverse/CEFCON/CEFCON.py]: The import will be error, will not use `from ..utils import data_preparation`?

File: omicverse/CEFCON/datasets.py [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+
+import os
+from pathlib import Path
+
+import pandas as pd
+import numpy as np
+from itertools import permutations, product
+import scanpy as sc
+import requests
+import zipfile
```
Comment (Starlitnightly) [inline omicverse/CEFCON/datasets.py]: error too.

**Review by Starlitnightly — CHANGES_REQUESTED** [review]
There are some import error should be updated.  And the data in the tutorial should be raw data, not processed data, and the raw data should be analysed using methods such as `ov.pp.qc` and `ov.pp.preprocess`. In addition, the graphs in the tutorial are a bit ugly, so please modify the plotting code if possible, and use the `ov.utils.embedding` function to visualise the umap.

Zehua

**Comment by DBinary** [comment]
These import error has been fixed and the tutorial of CEFCON will be updated soon.

**Pull request merged**